ts/src/bitmart.ts

```diff
@@ -1377,7 +1377,7 @@ export default class bitmart extends Exchange {
         //
         const data = this.safeValue (response, 'data', {});
         const trades = this.safeValue (data, 'trades', []);
-        return this.parseTrades (trades, market, since, limit);
+        return this.parseTrades (trades, market, since, limit, { 'takerOrMaker': 'taker' });
```

Comment:
isn't there an easier way other than having to extend it from here?

Reply:
I think it's easiest and more stable way, compared to handling it inside `parseTrade`. lmk if you have any other thoughts